src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/DetailedReportService.java

```diff
@@ -83,7 +83,9 @@ private void processReconciliationReport(
         }
 
         reconciliationService
-            .getSupplierStatement(date)
+            .getSupplierStatement(date) //XXX shouldn't this fetch by ID from the report? I
+            // with current implementation there is a slight chance that ReconciliationReport and detailedContent in it,
+            // will contain different info
```

Comment:
let remove this comment as it's not relevant to implemented feature

Reply:
Agreed it was just not for me when analysing the code, will remove, will ping Mustafa about his thoughts, but it doesn't seem correct to me, but maybe I am missing something.

Reply:
Removed now.